src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -202,7 +219,12 @@ public Task MessagesDeliveredAsync(IList<IBatchContainer> messages)
 
                 // clear cache and receiver
                 IEventHubQueueCache localCache = Interlocked.Exchange(ref cache, null);
-                EventHubReceiver localReceiver = Interlocked.Exchange(ref receiver, null);
+#if NETSTANDARD
+                PartitionReceiver
```

Comment:
@jason-bragg I'll revert this change now. I did the using but I found out that this Type is used later on in the same file and the `using` will not help since they use the same type but the property is in different places which is this case:

```
#if NETSTANDARD
                offset != PartitionReceiver.StartOfStream)
#else
                offset != EventHubConsumerGroup.StartOfStream) 
#endif
```
So, I prefer to stick with those `#if`s there and later on, when we decide to merge the trees we revisit it (which we will do anyway to remove the other `#if`s).